src/api.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://jennifers-first-web-service.onrender.com/api",
+});
+
+export const getArticles = () => {
+    return api.get(`/articles`).then(({ data }) => {
+            return data;
+          });
+        };
```

Comment:
Nice, but instead of sending the whole data object back we should ideally be sending the article information from this function instead